packages/component-submission/client/pages/SubmissionWizard.test.js

```diff
@@ -143,6 +144,34 @@ describe('SubmissionWizard', async () => {
       await testStepNavigation('/submit/id/disclosure', '/editors', 'back')
     })
 
+    it('scrolls to top on next', async () => {
+      const { getByTestId } = renderWithPath('/submit/id/author', pushHistory)
+
+      expect(scrollSpy).toBeCalledTimes(0)
+
+      fireEvent.click(getByTestId('next'))
+      await flushPromises()
+
+      expect(scrollSpy).toBeCalledTimes(1)
+      expect(scrollSpy).toBeCalledWith(0, 0)
+    })
+
+    it('scrolls to top on back', async () => {
+      const { getByTestId } = renderWithPath('/submit/id/files', pushHistory)
+
+      expect(scrollSpy).toBeCalledTimes(0)
```

Comment:
Assumption made here that we'll always be using `scrollTo` to scroll the user to the top of the window. Was a bit unhappy doing this as it's asserting implementation detail. However couldn't find a simple way of generically checking the scroll of window global. Let me know if you can think of anything. This does the job of adding coverage for now.